Source/WebKit/NetworkProcess/Downloads/cocoa/DownloadCocoa.mm

```diff
@@ -122,6 +128,62 @@
 #endif
     }
 }
+
+void Download::setPlaceholderURL(NSURL *placeholderURL)
```

Comment:
Nothing calls this function.  It seems like it should be asynchronous and something should be waiting for it before continuing.